sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -2253,6 +2536,58 @@ private void SetNameFieldsIfNull()
             }
         }
         #endregion UpdateMessage
+
+        #region Encoding
+        private static BinaryData? ToBinaryData(string input)
+        {
+            if (input == null)
+            {
+                return default;
+            } else
+            {
+                return new BinaryData(input);
+            }
+        }
+
+        private string EncodeMessageBody(BinaryData? binaryData)
+        {
+            if (!binaryData.HasValue)
+            {
+                return null;
+            }
+            switch (_messageEncoding)
+            {
+                case QueueClientOptions.MessageEncoding.None:
+                    return binaryData.Value.ToString();
+                case QueueClientOptions.MessageEncoding.Base64:
+                    return Convert.ToBase64String(binaryData.Value.Bytes.ToArray());
```

Comment:
Heads up that I think this was renamed from `.Bytes` to `.ToBytes()` in a later update.